docs/sources/dashboards/manage-dashboards/index.md

```diff
@@ -113,6 +113,25 @@ Changes are saved automatically.
 
 For more information about dashboard permissions, refer to [Dashboard permissions](ref:dashboard-permissions).
 
+## Delete multiple dashboards and folders
+
+When you delete one or more folders they are gone forever.
+However when you delete dashboards they will be moved to the Recently deleted section where they are stored for another 30 days. If you change your mind and want to get your dashboards back you can restore them easily.
+
+**To restore one or more dashboard:**
```

Comment:
Since this whole task is actually about restoring dashboards, this should be an H2 heading. Then we remove the bold from the task lead-in sentence because those shouldn't be bolded in general.
```suggestion
## Restore dashboards

To restore on or more dashboards, follow these steps:
```